### DISCUSSION TITLE
Git add yml file under .git/workflow for a branch is not working

### DISCUSSION THREAD

**Discussion by Dingjie-Daniel-Yang** [community/community#21417] [Other Feature Feedback, Questions, & Ideas]
<p>Hi Team,</p>
<p>I wanted to create a test.yml file under .git/workflows/test.yml to define an action for a local branch. When I tried to add the file to commit it, I was encountering the flowing error</p>
<pre><code class="lang-auto"> $git add .git/workflows/test.yml           
error: invalid path '.git/workflows/test.yml'
error: unable to add '.git/workflows/test.yml' to index
fatal: adding files failed
</code></pre>
<p>Any ideas as I am not sure what i missed for it?</p>
<p>Thanks</p>

**Comment by jsoref** [comment, marked as answer]
<p>You want <code>.github/workflows/test.yml</code> the <code>.git</code> directory is reserved for management by <code>git</code> itself. The GitHub metadata folder is <code>.github</code>.</p>

**Comment by Dingjie-Daniel-Yang** [comment]
<p>Thank you <a class="mention" href="/jsoref">@jsoref</a>. That makes sense.  It is working now :slight_smile:</p>
<p>Thank you<br>
Daniel</p>
